src/Makefile

```diff
@@ -85,7 +85,7 @@ ifeq ($(OS),macos)
 		LIBOMP_PREFIX:=$(shell brew --prefix libomp)
 		LIBOMP=$(LIBOMP_PREFIX)/lib/libomp.a
 	else
-		LIBOMP=-lomp
+		LIBOMP=-lomp -L$(shell brew --prefix libomp)/lib -Wl,-no_compact_unwind
```

Comment:
why this is not static link?

Reply:
I added it because it otherwise fails on macOS/M1 (it cannot find the libomp lib). We said we don't want static because it is unstable, no?